test/controllers/patients_controller_test.rb

```diff
@@ -33,8 +33,9 @@ class PatientsControllerTest < ActionDispatch::IntegrationTest
 
     it 'should not serve html' do
       sign_in @data_volunteer
-      get patients_path
-      assert_response :not_acceptable
+      assert_raise ActionController::UnknownFormat do
```

Comment:
I must have been extremely tired when I wrote this five days ago. It's okay as is since the point of the test is confirming it chokes on html. woof.